src/helpers/a11yAPI.ts

```diff
@@ -1,85 +1,157 @@
-// @flow
-import type { A11yRole, A11yStates, A11yState, A11yValue } from '../types.flow';
+import type { ReactTestInstance } from 'react-test-renderer';
+import type { AccessibilityRole, AccessibilityState } from 'react-native';
 import type { WaitForOptions } from '../waitFor';
 import makeA11yQuery from './makeA11yQuery';
 
+type AccessibilityStateKey = keyof AccessibilityState;
+type A11yValue = {
+  min?: number;
+  max?: number;
+  now?: number;
+  text?: string;
+};
+
 type GetReturn = ReactTestInstance;
 type GetAllReturn = Array<ReactTestInstance>;
 type QueryReturn = ReactTestInstance | null;
 type QueryAllReturn = Array<ReactTestInstance>;
 type FindReturn = Promise<GetReturn>;
 type FindAllReturn = Promise<GetAllReturn>;
 
-export type A11yAPI = {|
+export type A11yAPI = {
   // Label
-  getByA11yLabel: (string | RegExp) => GetReturn,
-  getByLabelText: (string | RegExp) => GetReturn,
-  getAllByA11yLabel: (string | RegExp) => GetAllReturn,
-  getAllByLabelText: (string | RegExp) => GetAllReturn,
-  queryByA11yLabel: (string | RegExp) => QueryReturn,
-  queryByLabelText: (string | RegExp) => QueryReturn,
-  queryAllByA11yLabel: (string | RegExp) => QueryAllReturn,
-  queryAllByLabelText: (string | RegExp) => QueryAllReturn,
-  findByA11yLabel: (string | RegExp, ?WaitForOptions) => FindReturn,
-  findByLabelText: (string | RegExp, ?WaitForOptions) => FindReturn,
-  findAllByA11yLabel: (string | RegExp, ?WaitForOptions) => FindAllReturn,
-  findAllByLabelText: (string | RegExp, ?WaitForOptions) => FindAllReturn,
+  getByA11yLabel: (label: string | RegExp) => GetReturn;
```

Comment:
Should we consider deprecating the aliases?

Reply:
I guess I would keep them for the time being, as not all a11y queries have their RTL-like counterpart.
@thymikee wdyt?

Reply:
In my ideal world each query should exists only once (no alias) and have the same behaviour as on web. For instance, we could drop `getByA11yLabel` and keep `getByLabelText`, since this one is supported on both platforms.

Reply:
Let's get rid of aliases for queries that are available on RTL, like ByLabelText for example. It's a simple search-replace change, so no biggie

Reply:
Removed the `*LabelText` and `*Role` aliases.